src/pages/basic_data/institution_admin/AddEmployeePage.tsx

```diff
@@ -113,7 +111,7 @@ export const AddEmployeePage = () => {
             <Box
               sx={{
                 display: 'grid',
-                gridTemplateColumns: { xs: '1fr', md: '1fr auto 1fr auto 1fr' },
+                gridTemplateColumns: { xs: '1fr', md: '3fr auto 2fr' },
```

Comment:
Dette sa du vel selv også, men ja, enig, det gir nok mer mening at begge panelene er like store her. Det er de for endre person også ser jeg nå
```suggestion
                gridTemplateColumns: { xs: '1fr', md: '1fr auto 1fr' },
```